src/_locales/ru/messages.yml

```diff
@@ -273,123 +273,125 @@ buttonInstallFromURL:
-  message: ''
+  message: Ошибка инициализации синхронизации.
 msgSyncReady:
   description: Message shown when sync will start soon.
-  message: ''
+  message: Синхронизация скоро начнется...
 msgSyncing:
   description: Message shown when sync is in progress.
-  message: ''
+  message: Синхронизация...
 msgSyncError:
   description: Message shown when sync failed.
-  message: ''
+  message: Ошибка синхронизации.
 msgNamespaceConflict:
   description: >-
     Message shown when namespace of the new script conflicts with an existent
     one.
-  message: ''
+  message: Конфликт пространства имен скриптов. Измените @name и @namespace.
 msgInvalidScript:
   description: Message shown when script is invalid.
-  message: ''
+  message: Неверный скрипт.
 labelShowBadge:
   description: Option to show number of running scripts on the badge.
-  message: ''
+  message: Показывать количество активных скриптов на значке.
 labelLineNumber:
   description: Label for line number jumper.
-  message: ''
+  message: Номер строки:
 labelBlacklist:
   description: Label for global blacklist settings in security section.
-  message: ''
+  message: Чёрный список
 descBlacklist:
   description: HTML Description for the global blacklist.
-  message: ''
+  message: Скрипты не будут вводиться в документы, URL адреса которых совпадают с правилами в этом списке.
 buttonSaveBlacklist:
   description: Button to save global blacklist.
-  message: ''
+  message: Сохранить
 msgSavedBlacklist:
   description: Message shown when blacklist are saved.
-  message: ''
+  message: Чёрный список сохранён.
 labelGeneral:
   description: Label for general settings.
-  message: ''
+  message: Общие настройки
 labelSyncScriptStatus:
   description: Label for option to sync script status.
-  message: ''
+  message: Синхронизировать статус скриптов.
 labelSyncDisabled:
   description: Label for option to disable sync service.
-  message: ''
+  message: ничем
 labelSyncService:
   description: Label for sync service select.
-  message: ''
+  message: Синхронизировать с
 labelSyncAuthorize:
   description: Label for button to authorize a service.
-  message: ''
+  message: Подключить
 labelSyncRevoke:
   description: Label for button to revoke authorization for a service.
-  message: ''
+  message: Отключить
 labelSyncAuthorizing:
   description: Label for button when authorization is in progress.
-  message: ''
+  message: Авторизация...
 msgSavedCustomCSS:
   description: Message shown when custom CSS is saved.
-  message: ''
+  message: Пользовотельские стили CSS сохранены.
 labelCustomCSS:
   description: Label for custom CSS section.
-  message: ''
+  message: Пользовотельские стили CSS
 descCustomCSS:
   description: Description of custom CSS section.
-  message: ''
+  message: >-
+    Пользовательские стили CSS для страницы Настройки и страницы установки скриптов. 
+	Если Вы не знаете что это такое, пожалуйста не редактируйте этот текст.
```

Comment:
Here is a syntax error. You may put them in one line if indentation is too much trouble.